ios/Classes/BetterPlayerPlugin.m

```diff
@@ -155,13 +157,18 @@ - (void) setRemoteCommandsNotificationActive{
 }
 
 - (void) setRemoteCommandsNotificationNotActive{
-    if ([_players count] == 0) {
```

Comment:
Memo:
After playing a video, `_players count` always > `0`. The Control center will always be linked till `AVAudioSession` got `setActive:false`.